packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -209,17 +209,17 @@ describe('<XGrid />', () => {
 
       const row = document.querySelector('[role="row"][aria-rowindex="2"]');
       const checkbox = row!.querySelector('input');
-      expect(row).to.not.have.class('Mui-selected');
+      expect(row).to.not.have.class('Mui-selected', 'class mui-selected 0');
       expect(checkbox).to.have.property('checked', false);
 
       fireEvent.click(screen.getByRole('cell', { name: 'Nike' }));
-      await raf();
-      expect(row!.classList.contains('Mui-selected')).to.equal(true);
+      await sleep(100)
+      expect(row!.classList.contains('Mui-selected')).to.equal(true, 'class mui-selected 1');
       expect(checkbox).to.have.property('checked', true);
 
       fireEvent.click(screen.getByRole('cell', { name: 'Nike' }));
       await raf();
-      expect(row!.classList.contains('Mui-selected')).to.equal(false);
+      expect(row!.classList.contains('Mui-selected')).to.equal(false, 'class mui-selected 2');
```

Comment:
Like in Jest we don't use assertion descriptions. It's very often outdated and doesn't help much.

```suggestion
      expect(row!.classList.contains('Mui-selected')).to.equal(false);
```

Reply:
this is to see where the fails in the ci

Reply:
Yes, I thought you where using Jest in the past, this isn't supported in Jest :p

Reply:
Happy to keep it, my feedback, at its core, is that it's often ignored and get outdated after refactoring in the tests. So usually, we don't bother 🤷‍♂️